x/ccv/provider/keeper/msg_server.go

```diff
@@ -448,6 +570,28 @@ func (k msgServer) UpdateConsumer(goCtx context.Context, msg *types.MsgUpdateCon
 		}
 	}
 
+	// add Owner event attribute
+	eventAttributes = append(eventAttributes, sdk.NewAttribute(types.AttributeConsumerOwner, currentOwnerAddress))
+
+	// add Phase event attribute
+	phase := k.GetConsumerPhase(ctx, consumerId)
+	eventAttributes = append(eventAttributes, sdk.NewAttribute(types.AttributeConsumerSpawnTime, phase.String()))
```

Comment:
Similar to previous comment, why `phase.String()` here when we talk about the spawn time?

Reply:
Same, it should be AttributeConsumerPhase.